.env.example

```diff
@@ -0,0 +1,2 @@
+JWT_SECRET="LocalSuperSecret"
+NODE_ENV=local
```

Comment:
A node env mint változó konvenció szerint 2 értéket vehet fel: 'development' és 'production' és ez nem azt jelenti, hogy milyen teszt környezeten fut, hanem azt, hogy az alkalmazás milyen módban fut (lokális futtatás esetén a development mód, erre hivatkozva tudunk bekapcsolni mondjuk extra debugging feature-öket, éles futtatás esetén pedig production mód) ebből adódik hogy a development meg a staging meg a production szerver node_env-je is konvenció szerint production.
Ha el akarjuk különíteni a teszt környezetet arra egy másik env változó bevezetése javasolt